src/models/room.js

```diff
@@ -189,6 +189,17 @@ export function Room(roomId, client, myUserId, opts) {
 
     if (this._opts.pendingEventOrdering == "detached") {
         this._pendingEventList = [];
+        const serializedPendingEventList = client._sessionStore.store.getItem(pendingEventsKey(this.roomId));
+        if (serializedPendingEventList) {
+            const self = this;
```

Comment:
You should just be able to use `this` since we have arrow functions now? (which indeed is what you've used)